sdk/storage/azure-storage-blobs/src/blob_sas_builder.cpp

```diff
@@ -134,53 +134,58 @@ namespace Azure { namespace Storage { namespace Blobs {
         + resource + "\n" + snapshotVersion + "\n" + CacheControl + "\n" + ContentDisposition + "\n"
         + ContentEncoding + "\n" + ContentLanguage + "\n" + ContentType;
 
-    std::string signature
-        = Base64Encode(Details::HmacSha256(stringToSign, Base64Decode(credential.GetAccountKey())));
+    std::string signature = Base64Encode(
```

Comment:
What's the plan for things in storage-common like `Base64Encode`. Such utility types shouldn't be public and probably belong to `::Details` as well.

Reply:
customers still want to use base64 encoding/decoding functions for MD5/CRC64

Reply:
So the plan is we don't